anchore_engine/services/policy_engine/engine/policy/gates/vulnerabilities.py

```diff
@@ -948,15 +948,13 @@ def evaluate(self, image_obj, context):
 
 class UnsupportedDistroTrigger(BaseTrigger):
     __trigger_name__ = "vulnerability_data_unavailable"
-    __description__ = (
-        "Triggers if vulnerability data is unavailable for the image's distro."
-    )
+    __description__ = "Triggers if vulnerability data is unavailable for the image's distro packages such as rpms or dpkg. Non-OS packages like npms and java are not considered in this evaluation"
 
     def evaluate(self, image_obj, context):
         if not have_vulnerabilities_for(DistroNamespace.for_obj(image_obj)):
             self._fire(
-                msg="Feed data unavailable, cannot perform CVE scan for distro: "
-                + str(image_obj.distro_namespace)
+                msg="Distro-specific feed data not found for distro namespace: %s. Cannot perform CVE scan OS/distro packages"
+                % image_obj.distro_namespace
```

Comment:
Looks good to me, a small nit around string format, should we start to replace the old string % operator with the new style format method.